pom.xml

```diff
@@ -563,21 +563,18 @@
                 <groupId>org.sonatype.aether</groupId>
                 <artifactId>aether-api</artifactId>
                 <version>1.13.1</version>
-                <scope>provided</scope>
             </dependency>
 
             <dependency>
                 <groupId>io.airlift.resolver</groupId>
                 <artifactId>resolver</artifactId>
                 <version>1.5</version>
-                <scope>provided</scope>
```

Comment:
aether-api, resolver and asm only used in local development mode, used to load the plug-in through maven to load the plug-in pom.xml file configured in alert.properties to facilitate debugging during development. Not required when the server deployment is running.